.octopus/deployment_settings.ocl

```diff
@@ -6,7 +6,7 @@ connectivity_policy {
 versioning_strategy {
 
     donor_package {
-        step = "Push" 
-        package = "Octopus.Ocl"
+        package = "NuGetPush.Source.Package"
+        step = "Push"
```

Comment:
Weird, `step` should be `push` (lowercase) here. Seems to work in Octopus though

Reply:
The slug is "push" in the portal.